server/routerlicious/packages/routerlicious-base/src/alfred/runnerFactory.ts

```diff
@@ -264,89 +260,163 @@ export class AlfredResourcesFactory implements core.IResourcesFactory<AlfredReso
+		);
+		restTenantThrottlers.set(
+			Constants.submitOpsThrottleIdPrefix,
+			socketSubmitOpTenantThrottler,
+		);
 
 		// Per-cluster Rest API Throttlers
 		const restApiCreateDocThrottleConfig = utils.getThrottleConfig(
-			config.get("alfred:throttling:restCallsPerCluster:createDoc"),
+			config.get("alfred:throttling:generalCluster:createDoc"),
 		);
 		const restCreateDocThrottler = configureThrottler(restApiCreateDocThrottleConfig);
-
 		const restApiGetDeltasThrottleConfig = utils.getThrottleConfig(
-			config.get("alfred:throttling:restCallsPerCluster:getDeltas"),
+			config.get("alfred:throttling:generalCluster:getDeltas"),
 		);
 		const restGetDeltasThrottler = configureThrottler(restApiGetDeltasThrottleConfig);
-
 		const restApiGetSessionThrottleConfig = utils.getThrottleConfig(
-			config.get("alfred:throttling:restCallsPerCluster:getSession"),
+			config.get("alfred:throttling:generalCluster:getSession"),
 		);
 		const restGetSessionThrottler = configureThrottler(restApiGetSessionThrottleConfig);
+		const socketConnectionThrottleConfigPerCluster = utils.getThrottleConfig(
+			config.get("alfred:throttling:generalCluster:socketConnections"),
+		);
+		const socketConnectClusterThrottler = configureThrottler(
+			socketConnectionThrottleConfigPerCluster,
+		);
+		const submitOpThrottleConfig = utils.getThrottleConfig(
+			config.get("alfred:throttling:generalCluster:submitOps"),
+		);
+		const socketSubmitOpThrottler = configureThrottler(submitOpThrottleConfig);
+		const submitSignalThrottleConfig = utils.getThrottleConfig(
+			config.get("alfred:throttling:generalCluster:submitSignals"),
+		);
+		const socketSubmitSignalThrottler = configureThrottler(submitSignalThrottleConfig);
 
 		const restClusterThrottlers = new Map<string, core.IThrottler>();
 		restClusterThrottlers.set(Constants.createDocThrottleIdPrefix, restCreateDocThrottler);
 		restClusterThrottlers.set(Constants.getDeltasThrottleIdPrefix, restGetDeltasThrottler);
 		restClusterThrottlers.set(Constants.getSessionThrottleIdPrefix, restGetSessionThrottler);
-
-		// Socket Connection Throttler
-		const socketConnectionThrottleConfigPerTenant = utils.getThrottleConfig(
-			config.get("alfred:throttling:socketConnectionsPerTenant"),
+		restClusterThrottlers.set(
+			Constants.socketConnectionsThrottleIdPrefix,
+			socketConnectClusterThrottler,
 		);
-		const socketConnectTenantThrottler = configureThrottler(
-			socketConnectionThrottleConfigPerTenant,
+		restClusterThrottlers.set(Constants.submitOpsThrottleIdPrefix, socketSubmitOpThrottler);
+		restClusterThrottlers.set(
+			Constants.submitSignalThrottleIdPrefix,
+			socketSubmitSignalThrottler,
 		);
 
-		const socketConnectionThrottleConfigPerCluster = utils.getThrottleConfig(
-			config.get("alfred:throttling:socketConnectionsPerCluster"),
+		// Tenant Group 1 Rest API Throttlers
+		const restApiCreateDocThrottleConfigGroup1 = utils.getThrottleConfig(
+			config.get("alfred:throttling:generalCluster:createDoc"),
```

Comment:
All of the configs from here and below should be using `tenantGroup1` instead of `generalCluster`, I think